src/cache/cache_las.c

```diff
@@ -754,22 +754,23 @@ __wt_las_insert_updates(WT_CURSOR *cursor, WT_BTREE *btree, WT_PAGE *page, WT_MU
                  * updates in the middle.
                  */
                 nentries = MAX_REVERSE_MODIFY_NUM;
```

Comment:
Why don't we skip the updates flagged when we push them into stack? We can push the first flagged updates on stack and break.

Reply:
Then when we pop the first update on stack, we can check if it is flagged. If it is flagged, we get the full value from lookaside and skip inserting it to lookaside. The rest of the code should remain the same. This is how I think it should work.

Reply:
Your solution requires every change in lookaside to be flagged and we still have to apply forward deltas to construct the full values of each update.